corehq/apps/cloudcare/static/cloudcare/js/formplayer/utils/utils.js

```diff
@@ -264,15 +264,23 @@ hqDefine("cloudcare/js/formplayer/utils/utils", function () {
             this.sortIndex = null;
         };
 
-        this.setQueryData = function ({ inputs, execute, forceManualSearch}) {
+        this.setQueryData = function ({ inputs, execute, forceManualSearch, initiatedBy}) {
```

Comment:
I notice this function calls it `initiatedBy`, but others call it `initiator` - could you unify the terminology for clarity?

Reply:
Changed to `initiatedBy` at other places